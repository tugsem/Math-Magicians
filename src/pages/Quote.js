import React from 'react'

const Quote = () => {
    return (
        <div className='quote'>
        <h1>Without mathematics, there's nothing you can do. Everything around you is mathematics. Everything around you is numbers.
        </h1>
        <p id='author'>-Shakuntala Devi</p>
        </div>
    )
}

export default Quote