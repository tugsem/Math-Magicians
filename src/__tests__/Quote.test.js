import renderer  from 'react-test-renderer';
import Quote from '../pages/Quote';

it('Renders correctly',()=>{
    const tree = renderer.create(
        <Quote />).toJSON();
        expect(tree).toMatchSnapshot();
})