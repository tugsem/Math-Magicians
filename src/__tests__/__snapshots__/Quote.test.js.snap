// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div
  className="quote"
>
  <h1>
    Without mathematics, there's nothing you can do. Everything around you is mathematics. Everything around you is numbers.
  </h1>
  <p
    id="author"
  >
    -Shakuntala Devi
  </p>
</div>
`;
